### PR TITLE
Feature/activity executions

### PR DESCRIPTION
**Pull request by sihu**
Open Tasks:

- [x] Hard coded translations to de.yml (@jimmylevell )
- [x] If needed: translate the new translations of this PR in fr.yml
- [x] Error management: Show errors inside editor (@jimmylevell)
- [x] Remove `overlap=false` from fixedEvents (@jimmylevell)
- [x] Change: Copy should open editor instead of copying directly (@jimmylevell)
- [ ] Check with @lukaseisenring to adapt the deployment with new ENVs (@diegosteiner)
- [x] Explaining video (maybe @diegosteiner ?)

Can be done later:
- [ ] Assisting views to show capacity and need of created executions
- [ ] Validations to check if a activity execution is valid (Field-Capacity, Spot limit?)
- [ ] More Calendar views: per Field, per spot
- [ ] Spot: add a train station

### CODE REVIEW

File: app/webpacker/components/CalendarManager.js [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+            <MenuItem size="small"
+                      onClick={() => this.handleEventRemove(eventInfo.event.id)}><DeleteIcon/>Delete</MenuItem>
+          </Menu>
+        </div>
+      </>
+    )
+  }
+
+  // render top information div displaying current events
+  renderSidebar() {
```
[the remark this answers is not in the document]
Comment (jimmylevell) [reply app/webpacker/components/CalendarManager.js:315]: Totally agree. As requested I have removed that function completely.

**Comment by jimmylevell** [comment]
as promised I have implemented my parts and pushed it. 

-  Hard coded translations to de.yml (@jimmylevell )
--> checked all translations, added missing and ensured same wording "e.g Activity execution"

-  If needed: translate the new translations of this PR in fr.yml
--> created a draft of the translation

-  Error management: Show errors inside editor (@jimmylevell)
--> the error message from the backend is now being displayed everywhere. 

-  Remove overlap=false from fixedEvents (@jimmylevell)
--> implemented, renamed the property overlap to fixedEvent

-  Change: Copy should open editor instead of copying directly (@jimmylevell)
--> now simply open the editor with the values of the "copied" event. The event is not effectively created. This has to be done by pressing "safe" in the editor. So simply open editor with prefilled values.

Ps. can we remove the HelloWorld React component?

File: app/models/activity_execution.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class ActivityExecution < ApplicationRecord
+  include Bitfields
+  validates_with StartBeforeEndValidator
+
+  belongs_to :activity, inverse_of: :activity_executions
+  belongs_to :field, inverse_of: :activity_executions
+  has_one :spot, through: :field
+  validates_presence_of :starts_at, :ends_at, :transport
+  validates :language_flags, numericality: { greater_than: 0 }, allow_nil: false
```
[the remark this answers is not in the document]
Comment (diegosteiner) [reply app/models/activity_execution.rb:12]: :+1:

**Comment by diegosteiner** [comment]
Einige Punkte die mir aufgefallen sind:

[ ] Validierung Feld und Programmspot: Wenn nichts ausgewählt wird nicht validiert, sondern eine Fehlermeldung (Bad Request)
[ ] Anzahl TN defaultmässig aus der Aktivität übernehmen
[ ] Fixpunkte allenfalls auch mit einem Feld versehen?

**Comment by jimmylevell** [comment]
- [ ] Improve time and date view in editor

**Pull request merged**